image-loader/src/desktopTest/kotlin/com/seiko/imageloader/screenshot/ChangeImageUrlTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.seiko.imageloader.screenshot
+
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onRoot
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.runComposeUiTest
+import com.github.takahirom.roborazzi.ExperimentalRoborazziApi
+import com.github.takahirom.roborazzi.InternalRoborazziApi
+import com.github.takahirom.roborazzi.RoborazziContext
+import com.github.takahirom.roborazzi.RoborazziOptions
+import io.github.takahirom.roborazzi.captureRoboImage
+import org.junit.Before
+import org.junit.Test
+
+class ChangeImageUrlTest : ChangeImageUrlCommonTest() {
+
+    @OptIn(ExperimentalRoborazziApi::class, InternalRoborazziApi::class)
+    @Before
+    fun initRoborazziConfig() {
+        RoborazziContext.setRuleOverrideOutputDirectory(
+            outputDirectory = "src/desktopTest/snapshots/images",
+        )
+    }
+
+    @OptIn(ExperimentalTestApi::class)
+    @Test
+    fun test_image_change() = runComposeUiTest {
```

Comment:
You may need to use runDesktopComposeUiTest intead of runComposeUiTest. Because it seems that runComposeUiTest doesn't provide rendering.